docs/guides/react-native.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: React Native
+description: Using Jotai in React Native
+nav: 3.5
+---
+
+Jotai atoms can be used in React Native applications with very few changes except for the use of persistence.
+
+Jotai has an [atomWithStorage function in the utils bundle](../api/utils.mdx#atom-with-storage) for persistance that supports persisting state in `sessionStorage`, `localStorage`, `AsyncStorage`, or the URL hash.
+
+Because react-native only supports AsyncStorage, you need to tell Jotai to use AsyncStorage instead of the default.
+
+Then, depending on what you are storing, you may need to take additional steps.  By default, AsyncStorage stores strings.  So, if you are simply storing a string, just passing AsyncStorage in is enough.
+
+## A simple pattern with AsyncStorage
+
+```js
+export const strAtom = atomWithStorage('string', undefined, AsyncStorage)
```

Comment:
Ah, I never knew you can do this. We don't do this with `localStorage` because that will break SSR. But, RN doesn't have SSR, it might be okay.